docs/understand/enterprise/fluffy-configuration.md

```diff
@@ -0,0 +1,128 @@
+    + `auth.saml.idp_skip_verify_tls_cert` `(bool : false)` - Insecure skip verification of the IDP TLS certificate, like when signed by a private CA
+    + `auth.saml.idp_authn_name_id_format` `(string : 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified')` - The format used in the NameIDPolicy for authentication requests
+    + `auth.saml.idp_request_timeout` `(duration : '10s')` The timeout for remote authentication requests.
+    + `auth.saml.external_user_id_claim_name` `(string : '')` - The claim name to use as the user identifier with an IDP mostly for logout
+  + `auth.oidc` Configuration section for OIDC
+    + `auth.oidc.enabled` `(bool : false)` - Enables OIDC Authentication.
+    + `auth.oidc.url` `(string : '')` - The OIDC provider url, e.g. 'https://oidc-provider-url.com/'
+    + `auth.oidc.client_id` `(string : '')` - The application's ID.
+    + `auth.oidc.client_secret` `(string : '')` - The application's secret.
+    + `auth.oidc.callback_base_url` `(string : '')` - A default callback address of the Fluffy server.
+    + `auth.oidc.callback_base_urls` `(string[] : '[]')`
+  + **Note:** You may configure a list of URLs that the OIDC provider may redirect to. This allows lakeFS to be accessed from multiple hostnames while retaining federated auth capabilities.
+    If the provider redirects to a URL not in this list, the login will fail. This property and callback_base_url are mutually exclusive.
+    {: .note }
+    + `auth.oidc.authorize_endpoint_query_parameters` `(bool : map[string]string)` - key/value parameters that are passed to a provider's authorization endpoint.
+    + `auth.oidc.logout_endpoint_query_parameters` `(string[] : '[]')` - The query parameters that will be used to redirect the user to the OIDC provider after logout, e.g. '[returnTo, https://<lakefs.ingress.domain>/oidc/login]'
+    + `auth.oidc.logout_client_id_query_parameter` `(string : '')` - The claim name that represents the client identifier in the OIDC provider
+    + `auth.oidc.additional_scope_claims` `(string[] : '[]')` - Specifies optional requested permissions, other than `openid` and `profile` that are being used.
+  + `auth.cache` Configuration section for RBAC service cache
+    + `auth.cache.enabled` `(bool : true)` - Enables RBAC service cache
+    + `auth.cache.size` `(int : 1024)` - Number of users, policies and credentials to cache.
+    + `auth.cache.ttl` `(duration : 20s)` - Cache items time to live expiry.
+    + `auth.cache.jitter` `(duration : 3s)` - Cache items time to live jitter.
+* `external` - Configuration section for the external authentication methods
+  + `external.aws_auth` - Configuration section for authenticating to lakeFS using AWS presign get-caller-identity request: [External Principals AWS Auth]({% link reference/security/external-principals-aws.md %})
```

Comment:
Please see [fluffy server configuration reference](https://docs.lakefs.io/reference/security/external-principals-aws.html#server-configuration), **regarding all the external auth related fields** - it's duplicate maintenance that im not sure we will remember to update. 
It was written initially because the page you added now was missing. 
Maybe delete it?